backend/webapi/Features/ClientLogs/Create.cs

```diff
@@ -0,0 +1,50 @@
+namespace Pidp.Features.ClientLogs;
+
+using FluentValidation;
+using Pidp.Data;
+using Pidp.Models;
+
+public class Create
+{
+    public class Command : ICommand
+    {
+        public string Message { get; set; } = string.Empty;
+
+        public LogLevel? LogType { get; set; }
+
+        public string BrowserInformation { get; set; } = string.Empty;
```

Comment:
I'm not sure about this property here, these are theoretically "generic" client logs; I don't think having a required field called `BrowserInformation` works for us.